percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.25-25.12.rst

```diff
@@ -0,0 +1,69 @@
+
+ ``mysqldump`` SST could stall due to a regression in desync mutex introduced in |Percona XtraDB Cluster| :rn:`5.6.24-25.11` by fixing the bug :bug:`1288528`. Bug fixed PXC-423.
+
+ ``mysql_tzinfo_to_sql`` sets :variable:`wsrep_replicate_myisam` variable at session scope so that statements are replicated correctly. Bug fixed PXC-332.
+
+ ``Percona-XtraDB-Cluster-devel-56`` package was not included in the ``Percona-XtraDB-Cluster-full-56`` metapackage on *CentOS* due to a conflict with upstream ``mysql`` package. Bug fixed PXC-381.
+
+ Running ``service mysql start`` and then ``service mysql@boostrap start`` afterwards would cause server shutdown. Bug fixed PXC-385.
+
+ ``NO_WRITE_TO_BINLOG`` / ``LOCAL`` for ``ANALYZE TABLE``, ``OPTIMIZE TABLE``, ``REPAIR TABLE``, ``FLUSH`` commands will ensure it is not written to binary log (as in mysql async replication) and not replicated in wsrep. Bug fixed PXC-391. 
+
+ ``FLUSH LOCAL TABLES WITH READ LOCK`` failure (with non-existent tables) didn't resume the galera provider, causing deadlock. Bug fixed PXC-399.
+
+ |Percona XtraDB Cluster| will not blocking DDL statements on tables which are used with ``... FOR EXPORT`` or ``... WITH READ LOCK``, it will give return an error message about read lock. Bug fixed PXC-403.
+
+ Fixed the update of :variable:`variable wsrep_slave_threads` variable regarding the default value assignment, invalid value truncation, and error issued while threads are still being closed. Bug fixed PXC-420.
+
+ Server would crash during startup if :variable:`gcs.fc_limit` variable was specified twice in :variable:`wsrep_provider_options`. Bug fixed PXC-428.
+
+ The mysql client in |Percona XtraDB Cluster| has been built with ``readline`` instead of ``editline``. Bug fixed PXC-430.
+
+ Bugs in 32-bit galera associated with ``statvfs`` in ``available_storage`` and integer overflow after multiplication in offset calculation have been fixed. Bug fixed PXC-433.
+
+ Galera 3 was failin to build on all non-intel platforms. Bug fixed PXC-326.
+
+ Architecture specific ``CCFLAGS`` have been removed and provision for inheriting ``CCFLAGS``, ``CFLAGS``, ``CXXFLAGS`` and ``LDFLAGS`` have been added to ``SConstruct``.
+
+ Non-global read locks such as ``FLUSH TABLES WITH READ LOCK`` and ``FLUSH TABLES FOR EXPORT`` paused galera provider but didn't block commit globally which :variable:`wsrep_to_isolation_begin` (for DDL) was made aware of. Bug fixed PXC-403.
+
+Following bug fixes have been ported from |Percona Server| `5.6.26-74.0 <https://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.26-74.0.html)>`_: bug :bug:`1454441`, bug :bug:`1470677`, bug :bug:`1472256`, bug :bug:`1472251`, and bug :bug:`1464468`.
```

Comment:
Since 
```
Percona Server has temporarily disabled savepoints in triggers and stored functions. The reason is that even having fixed bug #1438990 and bug #1464468 we have found more cases where savepoints in triggers break binary logging and replication, resulting in server crashes and broken slaves. This feature will be disabled until the above issues are properly resolved.
``` 
is not there in that (and there seems to be no bug for it), can you add it in a  separate line as:
```
Percona XtraDB Cluster (and Percona Server) have temporarily disabled savepoints in triggers and stored functions.
```